tests/onnx/quantization/test_graphs.py

```diff
@@ -12,22 +12,18 @@
 """
 
 # pylint: disable=no-member, redefined-outer-name, no-name-in-module
-
 import pytest
 
 from tests.onnx.quantization.common import infer_model, min_max_quantize_model
 from tests.onnx.quantization.common import compare_nncf_graph
 from tests.onnx.models import ALL_SYNTHETIC_MODELS
 from tests.onnx.models import MultiInputOutputModel
-from tests.onnx.models import ReshapeWeightModel
 
 
 @pytest.mark.parametrize('model_cls_to_test', ALL_SYNTHETIC_MODELS.values())
 def test_syntetic_models_graph(model_cls_to_test):
     if model_cls_to_test == MultiInputOutputModel:
         pytest.skip('min_max_quantize_model does not support many inputs for now.')
-    if model_cls_to_test == ReshapeWeightModel:
-        pytest.skip('There is incorrect quantization of the weight tensor. Should be fixed in #1362 PR.')
```

Comment:
Why was this model removed? In the previous PR, there was a message to fix this.

Reply:
This model is not skipped. Skipping is removed for this model